packages/web-components/src/components/filter-panel/filter-panel-modal.ts

```diff
@@ -61,6 +121,80 @@ class DDSFilterPanelModal extends HostListenerMixin(
     );
   }
 
+  get focusableElements() {
```

Comment:
Do we need to copy pasta this method here? Isn't it inherited now that we extend `DDSExpressiveModal`?